File: python3/1 - Iniciante/1037.py
def checkIntervalo(x):
    x = float(x)
    # ([0, 25], (25, 50], (50, 75], (75, 100])
    if 0.0 <= x <= 25.0:
        print('Intervalo [0,25]')
    elif 25.0 < x <= 50.0:
        print('Intervalo (25,50]')
    elif 50.0 < x <= 75.0:
        print('Intervalo (50,75]')
    elif 75.0 < x <= 100.0:
        print('Intervalo (75,100]')
    else:
        print('Fora de intervalo')

n = input()
checkIntervalo(n)

# a = 2.2 * 3.0
# b = 3.3 * 2.0

# checkIntervalo(25.01)
# checkIntervalo(25.00)
# checkIntervalo(100.00)
# checkIntervalo(-25.02)

File: ruby/1 - Iniciante/1037.rb
require 'bigdecimal'

# puts  123.6 - 123
# x = BigDecimal("123.6") - BigDecimal("123")
# puts x.to_f

entrada = BigDecimal(gets.chomp.to_s)
if entrada >= 0
	if entrada <= 25.0
		puts "Intervalo [0,25]"
	elsif entrada <= 50.0
		puts "Intervalo (25,50]"
	elsif entrada <= 75.0
		puts "Intervalo (50,75]"
	elsif entrada <= 100.0
		puts "Intervalo (75,100]"
	else
		puts "Fora de intervalo"
	end
else
	puts "Fora de intervalo"
end

